### PR TITLE
Improve slack reporting

### PR DESCRIPTION
**Pull request by vojtech-cerveny**
This pull request includes several changes to the `.circleci/config.yml` file to enhance test result reporting and notification. The most important changes involve adding the installation of `xmllint`, parsing test results, and updating Slack notifications with test summaries.

Enhancements to test result reporting:

* [`.circleci/config.yml`](diffhunk://#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R33-R35): Added a step to install `xmllint` for parsing XML test results.
* [`.circleci/config.yml`](diffhunk://#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R59-R72): Added a step to parse test results using `xmllint` and export the results as environment variables.

Improvements to Slack notifications:

* [`.circleci/config.yml`](diffhunk://#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L67-R84): Updated the Slack notification for failed tests to include a summary of passed tests, failed tests, and errors.
* [`.circleci/config.yml`](diffhunk://#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L148-R165): Updated the Slack notification for passed tests to include a summary of passed tests.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -64,7 +81,7 @@ jobs:
                   "type": "section",
                   "text": {
                     "type": "mrkdwn",
-                    "text": " The Tidepool Web UI tests have failed. :sad_tapani: \n\n *Please confirm resolution details:*"
+                    "text": "*:x: Tidepool Web UI Tests Failed* :sad_tapani: \n\n *Summary:* \n *$PASSED_TESTS/$TOTAL_TESTS tests passed* \n *$FAILURES tests failed* \n *$ERRORS errors encountered* \n\n :link: *<${CIRCLE_BUILD_URL}|View CircleCI Build Details>* \n\n :exclamation: *Please confirm resolution details.*"
```
Comment (ginnyyadav) [inline .circleci/config.yml:84]: Can we remove the resolution details section +checkboxes all together? I originally though the checkboxes would work, but you technically need an api set up to receive the checkbox request and post it back to slack which seems overkill. I'm happy to add anything to make this more useful or we can just commit to commenting on the slack thread when the tests fail.

**Review by ginnyyadav — CHANGES_REQUESTED** [review]
One small favor if you could remove the resolution details section (explained in comment). Otherwise LGTM.

File: .circleci/config.yml (hunk 1 above)
Comment (vojtech-cerveny) [reply .circleci/config.yml:84]: Will do + should I tag someone in the message? You, Brian? Or some group?

File: tests/e2e/clinicianPageFunctional.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ module.exports = {
     clinicPatientList.expect.element('@GMI').text.to.contain('GMI');
     clinicPatientList.expect.element('@timeInRange').text.to.contain('% Time in Range');
     clinicPatientList.expect.element('@avgGlucose').text.to.contain('Avg. Glucose');
-    clinicPatientList.expect.element('@lows').text.to.contain('Lows');
+    clinicPatientList.expect.element('@lowss').text.to.contain('Lows');
```
Comment (ginnyyadav) [inline tests/e2e/clinicianPageFunctional.js]: Did you mean to do this?

**Review by ginnyyadav — APPROVED** [review]
Ignore my other comment. I saw the revert afterwards :). Much better thank you :)

**Pull request merged**